presto-docs/src/main/sphinx/release/release-0.290.rst

```diff
@@ -0,0 +1,124 @@
+* Improve ``element_at`` by avoiding pushdown of negative position for ``element_at`` for array.  :pr:`23479`
+* Improve GET `/v1/info/state` to return INACTIVE state until the resource group configuration manager is fully initialized. :pr:`23585`
+* Improve IcebergDistributedSmokeTestBase by making it abstract. :pr:`23580`
+* Upgrade avro to version 1.11.4. :pr:`23868`
+* Upgrade commons-codec to version 1.17.0. :pr:`23868`
+* Upgrade commons-compress to version 1.26.2. :pr:`23868`
+* Upgrade commons-io to version 2.16.0. :pr:`23794`
+* Upgrade commons-io to version 2.16.1. :pr:`23868`
+* Upgrade commons-lang3 to version 3.14.0. :pr:`23868`
+* Upgrade protobuf-java to version 3.25.5. :pr:`23797`
+* Upgrade protobuf-java-util to version 3.25.5. :pr:`23797`
+* Fix nullability of columns in information schema :pr:`23577`
+* Fix distinct operator for UUID type :pr:`23732`.
+* Upgrade to centralize dependency versions in the parent pom.xml from presto-cassandra/pom.xml for consistency and ease of maintenance.  :pr:`23818`
+
+Presto C++ (Native Execution) Changes
+______________________________________
+* Add test for logical type storage in Parquet files. :pr:`23388`
+* Fix `task.writer-count` and `task.partitioned-writer-count` configuration properties in Presto C++ for consistency with Presto. :pr:`23902`
+* Add a metric `presto_cpp.memory_pushback_expected_reduction_bytes` to track expected reduction in memory after a pushback attempt. :pr:`23872`
+* Add a new counter, presto_cpp.memory_pushback_reduction_bytes, to monitor the actual memory reduction achieved with each memory pushback attempt. :pr:`23813`
+* Allow setting chunk size in ChunkedSliceOutput, it can reduce memory in the cases of large number of small output buffer. :pr:`23707`
+* Add ``native_max_local_exchange_partition_count`` session property which maps to the max_local_exchange_partition_count velox query property introduced in facebookincubator/velox#11292 :pr:`23910`
+* Add session property: 'native_writer_flush_threshold_bytes' which specifies the minimum memory footprint size required to reclaim memory from a file writer by flushing its buffered data to disk. :pr:`23891 `
+* Remove session property ``native_join_spiller_partition_bits``. :pr:`23906 `
+* Add session property: 'native_max_page_partitioning_buffer_size' which specifies the maximum bytes to buffer per PartitionedOutput operator to avoid creating tiny SerializedPages. :pr:`23853`
+* Add session property: 'native_max_output_buffer_size' which specifies the maximum size in bytes for the task's buffered output. The buffer is shared among all drivers. :pr:`23853`
+* Migrated arbitrator configs to use the new string-based format. :pr:`23496 `
+* Merged FilterNode into TableScanNode where possible, reducing I/O and CPU; added $path and $bucket to split info, and fixed split counts in coordinator UI. `:pr:23755`
```

Comment:
```suggestion
* Improve I/O and CPU usage by merging FilterNode into TableScanNode where possible. Also, added ``$path`` and ``$bucket`` to split info, and fixed the split counts in the coordinator UI. :pr:`23755`
```
Fix formatting of PR tag. 

Suggested revision to follow the [Order of changes](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes) in the Release Notes Guidelines. 

Note: Please review my suggested wording for accuracy - I read the original PR and think I described it correctly, but I want someone else to review this.